src/rpc/blockchain.cpp

```diff
@@ -460,7 +449,7 @@ UniValue getrawmempool(const JSONRPCRequest& request)
 
 UniValue getmempoolancestors(const JSONRPCRequest& request)
 {
-    if (request.fHelp || request.params.size() < 1 || request.params.size() > 2) {
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 2)
```

Comment:
If the guidelines say to put it onto the same line, we should modify the guidelines in this case...because I fully agree, that would look horrible.

Just re-read the guidelines, it says "it can" and not "it must", so all good :)